src/planners/PlannerInfo.cpp

```diff
@@ -29,10 +29,10 @@ void PlannerInfo::addCostConvergence(const std::vector<float> &cost_convergence)
 		PlannerInfo::cost_convergence.emplace_back(cost_convergence[i]);
 }
 
-void PlannerInfo::addRoutineTime(int time, int idx)
+void PlannerInfo::addRoutineTime(float time, int idx)
```

Comment:
size_t

Fix the vector size before the for loop.

Reply:
Hmm, unfortunately, I don't know how many elements will be added to the vector.